scripts/update_castxml_version.py

```diff
@@ -90,11 +90,12 @@ def generate_cmake_variables(urls_and_shas):
       set(macosx_binary_url    "{macosx_binary_url}")
       set(macosx_binary_sha512 "{macosx_binary_sha512}")
 
-      set(win32_binary_url    "NA")  # Windows 32-bit binaries not available
-      set(win32_binary_sha512 "NA")
-
       set(win64_binary_url    "{win64_binary_url}")
       set(win64_binary_sha512 "{win64_binary_sha512}")
+
+      # See https://github.com/CastXML/CastXML-python-distributions/issues/5
```

Comment:
```suggestion
      # Package 64-bit binaries as the 32-bit ones.
      # See https://github.com/CastXML/CastXML-python-distributions/issues/5
```

I also suggest to close issue #5

Reply:
I've closed issue 5